src/cdomains/apron/linearTwoVarEqualityDomain.apron.ml

```diff
@@ -697,14 +796,14 @@ struct
       lincons
     in
     let get_const acc i = function
-      | (None, o) ->
+      | (None, o, d) ->
         let xi = Environment.var_of_dim t.env i in
-        of_coeff xi [(Coeff.s_of_int (-1), xi)] o :: acc
-      | (Some r, _) when r = i -> acc
-      | (Some r, o) ->
+        of_coeff xi [(Coeff.s_of_int (- (Z.to_int d)), xi)] o :: acc
+      | (Some (c,r), _,_) when r = i -> acc
+      | (Some (c,r), o, d) ->
         let xi = Environment.var_of_dim t.env i in
         let ri = Environment.var_of_dim t.env r in
-        of_coeff xi [(Coeff.s_of_int (-1), xi); (Coeff.s_of_int 1, ri)] o :: acc
+        of_coeff xi [(Coeff.s_of_int (- (Z.to_int d)), xi); (Coeff.s_of_int @@ Z.to_int c, ri)] o :: acc
```

Comment:
Along the lines of `int64`, it would be safer to avoid these intermediate `int`s. The conversion `Z -> Coeff` is hardly obvious though...
```ocaml
Coeff.s_of_mpqf (Mpqf.of_mpz (Z_mlgmpidl.mpz_of_z d))
```
There are a couple of places that already duplicate this chain, maybe this should be extracted somewhere (`GobApron.Coeff.s_of_z`) and reused.